src/main/java/au/org/sports/wrugby/scoreboard/ScorerController.java

```diff
@@ -58,6 +58,7 @@ public ModelAndView scorerHTML() {
         mav.addObject("preset1", preset1);
         mav.addObject("preset2", preset2);
         mav.addObject("preset3", preset3);
+        mav.addObject("presetOptions", preset1.getName() + "," + preset2.getName() + "," + preset3.getName());
```

Comment:
Can you do this as mav.addObject("presets", Arrays.asList(preset1, preset2, preset3))?  Then in the page I think you can reference the various elements using preset[0].name etc.  or simply iterate the list (preferred).  You should also be able to produce the radio buttons directly from the list.